packages/edit-post/src/components/header/index.js

```diff
@@ -39,34 +39,32 @@ function Header( {
 			tabIndex="-1"
 		>
 			<HeaderToolbar />
-			{ ! isPublishSidebarOpened && (
-				<div className="edit-post-header__settings">
-					<PostSavedState
-						forceIsDirty={ hasActiveMetaboxes }
-						forceIsSaving={ isSaving }
+			<div className="edit-post-header__settings">
+				{ ! isPublishSidebarOpened && ( <PostSavedState
```

Comment:
The `PostSavedState` component is not completely hidden by the slide-in `PostPublishPanel`, so we need to take care of this.

Reply:
Do we want to remove it from the DOM or just visually hide it? Should this comment be in the code?

Reply:
Also it'd be nice to me if the component was on a separate line from the paren; that was one of the tweaks I made before the force-push 😉 eg:

```suggestion
				{ ! isPublishSidebarOpened && (
					<PostSavedState
```